app/src/controllers/log/selectors.js

```diff
@@ -194,3 +195,25 @@ export const retryLinkSelector = createSelector(
     },
   }),
 );
+
+export const disablePrevItemLinkSelector = createSelector(
+  stepPaginationSelector,
```

Comment:
Parent level can be any of levels. I suggest adding special selector which calls appropriate pagination selector (use `itemsSelector` as an example)